crates/eframe/src/native/run.rs

```diff
@@ -276,7 +276,41 @@ mod glow_integration {
 
         // Conceptually this will be split out eventually so that the rest of the state
         // can be persistent.
-        gl_window: glutin::WindowedContext<glutin::PossiblyCurrent>,
+        gl_window: GlutinWindowContext,
+    }
+    struct GlutinWindowContext {
+        window: winit::window::Window,
+        gl_context: glutin::context::PossiblyCurrentContext,
+        gl_display: glutin::display::Display,
+        gl_surface: glutin::surface::Surface<glutin::surface::WindowSurface>,
+    }
+
+    impl GlutinWindowContext {
+        fn window(&self) -> &winit::window::Window {
+            &self.window
+        }
+        fn resize(&self, physical_size: winit::dpi::PhysicalSize<u32>) {
+            use glutin::surface::GlSurface;
+            self.gl_surface.resize(
+                &self.gl_context,
+                physical_size
+                    .width
+                    .try_into()
+                    .expect("physical size must not be zero"),
+                physical_size
+                    .height
+                    .try_into()
+                    .expect("physical size must not be zero"),
```

Comment:
I seem to recall some platforms reporting 0x0 when window is minimized, so I'd rather have a safe fallback here (e.g. falling back to `1`).

Reply:
eframe already checks if size is `> 0` and only then calls the resize fn with arguments, so it should be fine i think. 

> same file, line 708
```rust
       winit::event::WindowEvent::Resized(physical_size) => {
                                // Resize with 0 width and height is used by winit to signal a minimize event on Windows.
                                // See: https://github.com/rust-windowing/winit/issues/208
                                // This solves an issue where the app would panic when minimizing on Windows.
                                if physical_size.width > 0 && physical_size.height > 0 {
                                    running.gl_window.resize(*physical_size);
                                }
```